src/request.go

```diff
@@ -102,7 +102,19 @@ func insertDomainMetrics(eventType string, domain string, beanAttrVals []*beanAt
 	// Create an entity for the domain
 	var e *integration.Entity
 	var err error
-	if args.LocalEntity {
+	if args.RemoteMonitoring {
+		if args.ConnectionURL != "" {
+			e, err = i.Entity(fmt.Sprintf("%s:%s", domain, args.ConnectionURL), "jmx-domain")
```

Comment:
We could use the specifications from here to parse the ConnectionUrl https://docs.oracle.com/javase/7/docs/api/javax/management/remote/JMXServiceURL.html

Reply:
port can be omitted and uri part can be the only difference, so what it we use this part `[host[:port]][url-path]` as a replacement for host:port in this case?

Reply:
we should make sure, there is no issue if there are special url characters into the entityName. An alternative would be to get host:port by checking the ":" location and then remove everything after the first "/"